letsencrypt-win-simple/Scripts/ImportExchange.ps1

```diff
@@ -1,56 +1,95 @@
 param(
     [Parameter(Position=0,Mandatory=$true)]
-    [string]$NewCertThumbprint,
+    [string]$NewCertThumbprint,         # should be {5} from letsencrypt.exe
     [Parameter(Position=1,Mandatory=$true)]
-    [string]$ExchangeServices
+    [string]$ExchangeServices,          # comma-separated list of exchange services to import for
```

Comment:
Maybe we should make this a ValidateSet() with the possible values `'IMAP', 'POP', 'UM',  'IIS', 'SMTP', 'Federation', 'UMCallRouter'`

Reply:
I think that might be a bit out of scope as it would require updating with specific knowledge on Exchange. Keep in mind this script should never be run interactively.